src/_pytest/terminal.py

```diff
@@ -233,29 +238,30 @@ def get_location(self, config):
 
 
 class TerminalReporter:
-    def __init__(self, config, file=None):
+    def __init__(self, config: Config, file=None):
```

Comment:
`-> None`

Reply:
Is there a way to have mypy complain about it?